daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/Daffodil.scala

```diff
@@ -200,9 +220,10 @@ class Compiler private[sapi] () {
    *                   then no namespace is used. With not preceded by "{namespace}",
    *                   then Daffodil will figure out the namespace.
    */
+  @deprecated("Use DataProcessor.setExternalVariables", "2.6.0")
   def setExternalDFDLVariables(extVarsMap: Map[String, String]): Unit = {
```

Comment:
When I build the scaladoc,, for some reason this method doesnt show up as deprecated. Same issue with this function in the DataProcessor. And it's only this variable setter. All the other variable setters are marked as deprecated. Looks correct to me though...

Reply:
Probably bug in scaladoc. There are multiple overloads of this method name, and they even have the same number of arguments, same return tyhpe. Only the arg type is different. Easy to imagine a bug there.